### PR TITLE
Milestone 4 calculations

### PR DESCRIPTION
**Pull request by jcromerohdz**
Milestone - 4  calculations
- Preparation setup files (src/logic/ calculate.js and operate.js)
- Implementing the calculate.js module
- Implementing the operate.js module
- Add calculate.js to the App component

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import Big from 'big-js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const [firstNum, secondNum] = [Big(numberOne), Big(numberTwo)];
+
+  switch (operation) {
+    case '+':
+      return firstNum.plus(secondNum).toString();
+    case '-':
+      return firstNum.plus(secondNum).toString();
```
Comment (bellom) [inline src/logic/operate.js]: I guess you wanted to implement the `minus` operation here or up there right?

File: src/logic/calculate.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import operate from './operate';
```
Comment (bellom) [inline src/logic/calculate.js:1]: you should import this file in `App` component.
image

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @jcromerohdz  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/logic/operate.js (hunk 1 above)
Comment (jcromerohdz) [reply src/logic/operate.js]: You right !, thanks I will modify this

File: src/logic/calculate.js (hunk 2 above)
Comment (jcromerohdz) [reply src/logic/calculate.js:1]: My bad  I totally forgot this, sorry I will fix it and thanks again !

**Review by bellom — APPROVED** [review]
## Hi @jcromerohdz  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**